proto/buf/alpha/registry/v1alpha1/organization.proto

```diff
@@ -205,6 +206,10 @@ message UpdateOrganizationSettingsRequest {
   PluginRole plugin_base_role = 3;
   // optional, no update to this base role will be made if this is unspecified.
   TemplateRole template_base_role = 4;
+  // Changes are only made to values specified in field_mask.
+  google.protobuf.FieldMask field_mask = 5;
+  string description = 6;
+  string url = 7;
 }
```

Comment:
Note the last two fields look inconsistent with the ones above, as `repository_base_role` and the other two fields are not controlled by the field mask. I think this is fine because `organization` message doesn't have these roles as fields.